Source/Core/PltCtrlPoint.cpp

```diff
@@ -1242,6 +1242,12 @@ PLT_CtrlPoint::InspectDevice(const NPT_HttpUrl& location,
                              NPT_TimeInterval   leasetime)
 {
     NPT_AutoLock lock(m_Lock);
+	
+	if (!m_Started) {
+		NPT_LOG_INFO_1("InspectDevice called before control point started.  location url: %s",
```

Comment:
InspectDevice is called from ProcessSsdpMessage only. I am not sure I understand why it is important for the ctrl point to be started.